ci-operator/config/openshift/cluster-update-keys/openshift-cluster-update-keys-release-4.8__okd.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: origin
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```

Comment:
I thought so, too, but it's the one also used in OCP: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-update-keys/openshift-cluster-update-keys-release-4.8.yaml#L10
Maybe that should be upgraded?

Reply:
weird... thanks for the link

Reply:
@smarterclayton since you're the owner of this job: Should I go ahead update the build root for OCP 4.8 (and this OKD one) to a more current version of Go?

Reply:
I've updated this to 1.16 here, and opened a separate PR to bump the OCP 4.8 buildroot: https://github.com/openshift/release/pull/22422